servant/src/Servant/API.hs

```diff
@@ -83,6 +87,7 @@ import           Servant.API.ResponseHeaders (AddHeader (addHeader),
                                               HList (..), Headers (..),
                                               getHeadersHList, getResponse)
 import           Servant.API.Sub             ((:>))
+import           Servant.API.Times           (FTime(..), toProxy, getFormat, renderTime, parseTime)
```

Comment:
I can see it being useful when using the Time type, but it probably only belongs if the type were going to be used more broadly than just servant. Happy for it to be removed/to remove it.